sections/findify-filter-group.liquid

```diff
@@ -1,6 +1,6 @@
 
 {%- assign page_url = content_for_header | split:'"pageurl":"' | last | split: '"' | first | split: request.host | last | replace: '\/','/' | replace:'\u0026','&' | url_decode  -%}
-{%- assign filtersParam = page_url | split: "filters=" | last | split: '&' | first -%}
+{%- assign filtersParam = page_url | split: "filters=" | last | split: '&color_filter' | first -%}
```

Comment:
This &color_filter is not sustainable.
filtersParam shall have its own way to determine his value without relying on other query strings.

split: "filters=" | last | split: "&" | first should work

If it doesn't there is something else that must be changed.